app/docker/views/containers/create/createContainerController.js

```diff
@@ -612,6 +612,9 @@ angular.module('portainer.docker').controller('CreateContainerController', [
       Volume.query(
         {},
         function (d) {
+          d.Volumes.sort((vol1, vol2) => {
+            return vol1.Name.localeCompare(vol2.Name);
+          });
```

Comment:
not sure how to give a suggestion for more than a line.
instead of the change, I would suggest:
```
      $scope.availableVolumes =  d.Volumes.sort((vol1, vol2) => {
            return vol1.Name.localeCompare(vol2.Name);
          });
```

I know that Array.sort is in-place, but that can confuse us the next time we come to this code, or if we want to filter /map the results.

Reply:
Thank you for the feedback! I shall make the requested change.

Reply:
great!